docs/5-development/01-custom-UI5-Web-Components-Packages.md

```diff
@@ -294,4 +294,4 @@ import "my-ui5-webcomponents/dist/SomeOtherComponent.js";
 import "my-ui5-webcomponents/dist/YetAnotherComponent.js";
 ```
 
-Next: [Developing Custom UI5 Web Components](../custom-ui5-web-components)
+Next: [Developing Custom UI5 Web Components](./02-custom-UI5-Web-Components.md)
```

Comment:
This url seems to work
https://sap.github.io/ui5-webcomponents/playground/development/custom-ui5-web-components/